library_generation/test/resources/proto_path_list.txt

```diff
@@ -0,0 +1,15 @@
+# This file is used in integration test against `generate_library.sh`.
+# Format:
+# proto_path destination_path
+google/bigtable/v2 google-cloud-bigtable-v2-java
+google/cloud/apigeeconnect/v1 google-cloud-apigeeconnect-v1-java
+google/cloud/asset/v1 google-cloud-asset-v1-java
+google/cloud/compute/v1 google-cloud-compute-v1-java
+google/cloud/kms/v1 google-cloud-kms-v1-java
+google/example/library/v1 google-cloud-example-library-v1-java
+# google/iam/v1 google-iam-v1-java
```

Comment:
Temporarily exclude to generate `iam` due to #1992, will address the issue in another PR.